package.json

```diff
@@ -17,8 +17,9 @@
   "homepage": "https://docs.pi-hole.net/",
   "scripts": {
     "build": "mkdocs build --clean",
+    "build:strict": "npm run build -- --strict",
```

Comment:
We could also just move `--strict` in the `build` script and remove the new one.